src/components/Content/index.tsx

```diff
@@ -60,56 +67,57 @@ export const Content: React.VFC = () => {
 
   return (
     <PageGrid>
-      <Header />
       <Routes>
-        <Route path="*" element={<PageDoesNotExist />} />
-        <Route path="/" element={<MyPatches />} />
-        <Route path={routes.commits} element={<Commits />}>
-          <Route path=":id" element={null} />
-        </Route>
-        <Route path={routes.configurePatch} element={<ConfigurePatch />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route path={routes.host} element={<Host />} />
-        <Route path={routes.hosts} element={<Hosts />} />
-        <Route path={routes.jobLogs} element={<JobLogs />}>
-          <Route path=":groupId" element={null} />
-        </Route>
-        <Route path={routes.myPatches} element={<MyPatches />} />
-        <Route path={routes.patch} element={<VersionPage />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route path={routes.preferences} element={<Preferences />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route path={routes.projectPatches} element={<ProjectPatches />} />
-        <Route path={routes.projectSettings} element={<ProjectSettings />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route
-          path={routes.projectSettingsRedirect}
-          element={<ProjectSettingsRedirect />}
-        />
-        <Route path={`${routes.spawn}/*`} element={<Spawn />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route path={routes.commitQueue} element={<CommitQueue />} />
-        <Route path={routes.task} element={<Task />}>
-          <Route path={tab} element={null} />
-        </Route>
-        <Route path={routes.taskHistory} element={<TaskHistory />} />
-        <Route path={routes.taskQueue} element={<TaskQueue />}>
-          <Route path=":distro" element={null} />
-          <Route path=":distro/:taskId" element={null} />
-        </Route>
-        <Route path={routes.userPatches} element={<UserPatches />} />
-        <Route
-          path={routes.userPatchesRedirect}
-          element={<UserPatchesRedirect />}
-        />
-        <Route path={routes.variantHistory} element={<VariantHistory />} />
-        <Route path={routes.version} element={<VersionPage />}>
-          <Route path={tab} element={null} />
+        <Route element={<Layout />}>
+          <Route path="*" element={<PageDoesNotExist />} />
+          <Route path="/" element={<MyPatches />} />
+          <Route path={routes.commits} element={<Commits />}>
+            <Route path=":projectId" element={null} />
```

Comment:
Does this param actually refer to the project ID or the identifier? We've used them interchangeably so far, but EVG-16434 exists to address this. AFAIK most routes use the more user-friendly identifier but I may be mixed up.

Reply:
technically `identifier` (or maybe `projectIdentifier`?) is more correct haha 😅
I'll rename the paths, it's an easy fix (^^) thanks Sophie!